library/src/main/java/com/novoda/viewpageradapter/ViewPagerAdapterState.java

```diff
@@ -25,7 +25,7 @@ public static ViewPagerAdapterState newInstance() {
     }
 
     private static ViewPagerAdapterState from(Parcel in) {
-        Bundle bundle = in.readBundle();
+        Bundle bundle = in.readBundle(ViewPagerAdapterState.class.getClassLoader());
```

Comment:
this gets passed to the bundle, so when objects are created _from_ the bundle, it uses this class loader.